AddonBicepTemplate/AzureMonitorAlertsProfile.bicep

```diff
@@ -44,7 +44,7 @@ resource recommendedAlerts 'Microsoft.AlertsManagement/prometheusRuleGroups@2023
       }
       {
         alert: 'Average Memory usage per container is greater than 95%.'
-        expression: '(container_memory_working_set_bytes{container!="", image!="", container_name!="POD"} / on(namespace,cluster,pod,container) group_left kube_pod_container_resource_limits{resource="memory", node!=""}) > .95'
+        expression: 'sum by (namespace, controller, container, cluster)(((container_memory_working_set_bytes{container!="", image!="", container!="POD"} / on(namespace,cluster,pod,container) group_left kube_pod_container_resource_limits{resource="memory", node!=""})) * on(namespace, pod,  cluster) group_left(controller) label_replace(kube_pod_owner{owner_name=~"(.*)"}, "controller", "$1", "owner_name", "(.*)")) > .95'
```

Comment:
what does this do? I think nothing...
owner_name=~"(.*)"

Reply:
I agree, the "*" is not required since it does multiple character regex matching for owner_name. We can instead just use owner_name=~"(.)" according to our use case.